Frameworks/AEPSwiftUI/Sources/ContentCards/UIElement/AEPStack/AEPStack.swift

```diff
@@ -0,0 +1,61 @@
+/*
+ Copyright 2024 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+import AEPServices
+import Foundation
+import SwiftUI
+
+public class AEPStack: ObservableObject {
+    /// An array of child view models of the  stack, each conforming to `AEPViewModel`.
+    @Published var childModels: [any AEPViewModel] = []
+
+    /// The spacing between child views in the  stack.
+    @Published public var spacing: CGFloat?
+
+    /// Adds a view model to the stack.
+    /// This method is used internally to
+    /// - Parameter model: The view model to be added, conforming to `AEPViewModel`.
+    func addModel<M: AEPViewModel>(_ model: M) {
+        childModels.append(model)
+    }
```

Comment:
This function is used internally by the library to add UIElement's model into the Stack.
No need for removeModel API as internally while building templates we only add models.. no scenario to have a model removed